### ISSUE TITLE
[Feature Request] Get Posts from Specific Day/Week/Month across Years

### ISSUE THREAD

**Issue by mrgoff93** [ryanwelcher/advanced-query-loop#45]
Thanks for working on this plugin and sharing your work on its development. Although it was mostly well over my head (I have no experience with this sort of Wordpress development), I went through the 4 videos of your live streams and found it interesting to see how you worked. 

My blog is primarily a journal of nature observations that I've been writing and publishing for over 20 years now. I would like to be able to easily pull things together based on time of year and create something like an almanac based on a distillation of my observations.

To that end, I would like to be able to select posts that were published on a particular day (or week, or possibly month) across years. 

Since WP_Query() has date parameters for month (1-12), week (1-53), and day [of month] (1-31), I'm hoping it's fairly straightforward to do. 

I was thinking I might have a try at editing the example code, AQL-extension example code, but as someone who has never worked on this sort of thing, it may be well above my head.

Thanks

**Comment by ryanwelcher** [comment]
Thanks for using the plugin. If I understand correctly, you want to be able to select a day for every year? For example, a query that shows all posts from Jan 1 from any year?

**Comment by mrgoff93** [comment]
Yes - that's right. I would like to be able to select all the posts from a selected Month-Day regardless of year. 

I think of it as an "On this [selected] day in blog History" sort of option.

Thanks!

**Comment by carstingaxion** [comment]
Hello 👋🏻.

I had some similar intend and created a fork of @ryanwelcher|s repo to do that.

https://github.com/carstingaxion/additional-advanced-query-loops/assets/198883/8b9f071a-bc03-4063-8723-f59c7d153629

This may (not yet) be 100% of what you @mrgoff93 where looking for, but  please have a look at https://github.com/carstingaxion/additional-advanced-query-loops/issues/10#issuecomment-2130303912

**Comment by carstingaxion** [comment]
I hope we’ll find a way to merge all of this, because this now is possible with my [contextual-query fork](https://github.com/carstingaxion/additional-advanced-query-loops/issues/10#issuecomment-2146755716) and the issue _could be closed_ .

Have a look @mrgoff93 

![image](https://github.com/carstingaxion/additional-advanced-query-loops/assets/198883/80f98c48-cba9-4381-b80c-ed6b364cd665)

**Comment by ryanwelcher** [comment]
@carstingaxion thanks for all of your activity on this as of late! The best way to get this into AQL is via a pull request. You should be able to create one based on your fork.
